gff3tool/lib/gff3_merge/revision.py

```diff
@@ -20,7 +19,7 @@ def main(gff_file, revision_file, output_gff, report_file=None, user_defined1=No
 
     NCRNA = ['rRNA', 'miRNA', 'ncRNA', 'snRNA', 'snoRNA', 'tRNA']
 
-    logger.info('Reading revision file... ({0:s})'.format(revision_file))
+    logger.info('Reading revision file... (%s)'.format(revision_file))
     flines = open(revision_file, 'rb')
```

Comment:
When I run the gff3_merge example for this branch, I get the below error. This line is the only difference I can find between testpy3 and testpy3_rebase for revision.py and gff3_merge.py.

Error:
```
INFO     Generate auto_replace_tag/check1.txt for Check Point 1 internal reviewing...
INFO     Reading revision file... (%s)
Traceback (most recent call last):
  File "/app/data/monica_data/venv-python3/bin/gff3_merge", line 8, in <module>
    sys.exit(script_main())
  File "/app/data/monica_data/venv-python3/lib/python3.5/site-packages/gff3tool/bin/gff3_merge.py", line 229, in script_main
    main(args.gff_file1, args.gff_file2, args.fasta, report_fh, args.output_gff, args.all, args.auto_assignment, args.user_defined_file1, args.user_defined_file2, logger=logger_stderr)
  File "/app/data/monica_data/venv-python3/lib/python3.5/site-packages/gff3tool/bin/gff3_merge.py", line 70, in main
    gff3_merge.revision.main(gff_file=gff_file1, revision_file=autoFILE, output_gff=autoReviseGff, report_file=autoReviseReport, user_defined1=user_defined1, auto=auto, logger=logger)
  File "/app/data/monica_data/venv-python3/lib/python3.5/site-packages/gff3tool/lib/gff3_merge/revision.py", line 33, in main
    if not re.search('\t\n', line_raw):
  File "/app/data/monica_data/venv-python3/lib64/python3.5/re.py", line 173, in search
    return _compile(pattern, flags).search(string)
TypeError: cannot use a string pattern on a bytes-like object
```

Reply:
Oops, it seems to me that it's my mistake when doing rebase.